src/api/spec/components/bs_request_comment_component_spec.rb

```diff
@@ -2,49 +2,73 @@
 
 RSpec.describe BsRequestCommentComponent, type: :component do
   let(:commentable) { create(:bs_request_with_submit_action) }
-  let(:comment_a) { create(:comment_request, commentable: commentable, body: 'Comment A') }
+  let(:comment_a) { create(:comment_request, commentable: commentable, body: 'Comment A', created_at: Time.zone.yesterday) }
 
-  subject { render_inline(described_class.new(comment: comment_a, commentable: commentable, level: 1)) }
+  before do
+    render_inline(described_class.new(comment: comment_a, commentable: commentable, level: 1))
+  end
+
+  it 'displays a comment icon' do
+    expect(rendered_content).to have_selector('i.fa-comment', count: 1)
+  end
+
+  it 'displays an avatar' do
+    expect(rendered_content).to have_selector("img[title='#{comment_a.user.realname}']", count: 1)
+  end
+
+  it 'displays who wrote the comment' do
+    expect(rendered_content).to have_text("#{comment_a.user.realname} (#{comment_a.user.login})\nwrote")
+  end
+
+  it 'displays the time when the comment happened in words' do
+    expect(rendered_content).to have_text('1 day ago')
+  end
+
+  it 'displays the comment in a comment bubble' do
+    expect(rendered_content).to have_selector('.comment-bubble', text: 'Comment A')
+  end
```

Comment:
I wouldn't go that far as to test this. The bubble is some styling that can change in the future without affecting the feature.

Reply:
:+1:  change pushed